packages/devextreme/js/localization/messages/ar.json

```
@@ -310,6 +310,7 @@
         "dxTagBox-selected": "تم تحديد {0}",
         "dxTagBox-allSelected": "كل العناصر المحددة ({0})",
         "dxTagBox-moreSelected": "{0} المزيد",
+        "dxTagBox-tagRoleDescription": "tag. Press the delete button to remove this tag",
```

```suggestion
        "dxTagBox-tagRoleDescription": "Tag. Press the delete button to remove this tag",
```
Can you please capitalize "tag"?

---

[3c34bafb39088ac58375ca491e78bb36c2fc7666]